ambari-server/src/main/java/org/apache/ambari/server/audit/event/request/ChangeRepositoryVersionRequestAuditEvent.java

```diff
@@ -89,18 +89,16 @@ protected void buildAuditMessage(StringBuilder builder) {
         .append(repoVersion)
         .append("), Repositories(");
 
-      if (!repos.isEmpty()) {
-        builder.append(System.lineSeparator());
-      }
+
 
       for (Map.Entry<String, List<Map<String, String>>> repo : repos.entrySet()) {
         builder.append("Operating system: ").append(repo.getKey());
-        builder.append(System.lineSeparator());
+        builder.append("(");
         for (Map<String, String> properties : repo.getValue()) {
           builder.append("    Repository ID(").append(properties.get("repo_id"));
```

Comment:
I think the indentation can be removed:

```suggestion
           builder.append("Repository ID(").append(properties.get("repo_id"));
```

Reply:
Taken care